draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -202,8 +202,8 @@ normative:
 
 --- abstract
 
-This document specifies a number of algorithms that may be used to hash arbitrary
-strings to Elliptic Curves. 
+This document specifies a number of algorithms that may be used to encode an
+arbitrary string to a point on an Elliptic Curve.
```

Comment:
The title of the document uses "hashes," so this seems to be inconsistent. Perhaps we can say "encode or hash an arbitrary..."?